scripts/helpers/escape-string.js

```diff
@@ -1,5 +1,3 @@
-export const escapeString = {
-	escapeString(string) {
-		return string.replace(/([;&,.+*~':"!^#$%@[\]()=>|])/g, '\\$1');
-	},
-};
+export function	escapeString(string) {
```

Comment:
some weard spacing here...should this be like this?

Reply:
I think I've changed this 🙂